docs-ref-conceptual/LTS-version/azure-cli-sp-tutorial-1.md

```diff
@@ -30,6 +30,11 @@ In this tutorial, you learn how to:
 
 Use the [az ad sp create-for-rbac](/cli/azure/ad/sp#az_ad_sp_create_for_rbac) Azure CLI reference command to create a service principal. This example doesn't specify a `--name` parameter, so a name containing a time stamp is automatically created.
 
+> [!WARNING]
+> The `az ad sp create-for-rbac` command can modify an existing application or service principal if
```

Comment:
As stated in https://dev.azure.com/msft-skilling/Content/_workitems/edit/508461#5050393:

There is a separate command `az ad sp create` https://learn.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest#az-ad-sp-create to create service principal.

Actually, the problem with `az ad sp create-for-rbac` is that it updates the existing application. The problem also lies within `az ad app create`: https://github.com/Azure/azure-cli/blob/cf11272c36d2680a65bd775e10d338afa3a8b902/src/azure-cli/azure/cli/command_modules/role/custom.py#L609-L620